src/mock_display.rs

```diff
@@ -482,6 +532,48 @@ where
 
         display
     }
+
+    /// Checks if the displays are equal.
+    ///
+    /// An advanced output for failing tests can be enabled, see the [module-level documentation]
+    /// for more details.
```

Comment:
It can't hurt to mention this here as well IMO. One less link to click for the user, and this is the documentation that pops up when hovering over usages of `assert_eq` in VSCode, etc.

```suggestion
    /// An advanced output for failing tests can be enabled by setting the environment variable 
    /// `EG_FANCY_PANIC=1`. See the [module-level documentation] for more details.
```